app/Http/Controllers/ServiceController.php

```diff
@@ -194,6 +194,7 @@ public function update(Request $request, $id)
             }
 
             //If there are more qualifications than position: User has Add new Positions
+            $positions = array();
```

Comment:
This overwrites line 184 if this Service had positions before. Maybe we have to to like:

```
if(empty($positions)) {$positions = array();}
if(empty($qualifications)) {$qualifications= array();}
```

**OR**

`if (!empty($positions)) && !empty($qualifications) && count($positions) < count($qualifications))`